solutions/aspnet_core/aspnet_samples/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 as BUILDER
+
+# RUN apt update && apt install -y git vim
+
+# All project should use port 5000 for HTTP and 5001 for HTTPS (if exists)
+EXPOSE 5000 5001
+
+WORKDIR /aspnet
+RUN git clone --depth 1 --single-branch https://github.com/asvrada/samples.git
```

Comment:
We should include the sample code in this repo.

Reply:
Will do